google-cloud-datastore/samples/acceptance/tasks_test.rb

```diff
@@ -0,0 +1,91 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+require_relative "helper"
+require_relative "../tasks"
+
+describe "Datastore task list" do
+  let(:datastore) { Google::Cloud::Datastore.new }
+
+  before do
+    delete_tasks
+  end
+
+  def wait_until times: 5
+    times.times do
+      return if yield
+    end
```

Comment:
Maybe there should be a sleep between attempts? Are the uses of this stable enough?

Reply:
I agree, looks like an error in the original method. I'll add a 1 s sleep.

Reply:
Done.